README.md

```diff
@@ -3,6 +3,17 @@
 [![CircleCI](https://circleci.com/gh/singnet/snet-daemon.svg?style=svg)](https://circleci.com/gh/singnet/snet-daemon)
 
 SingularityNET Daemon
+The daemon is the adapter with which an otherwise SingularityNET-unaware service implementation can be exposed to the SingularityNET platform. It is designed to be deployed as a sidecar proxy alongside the service on a given host.
+The daemon abstracts the blockchain components away from the clients.
+The SNET Daemon interacts with the Multi Party Escrow to facilitate authorization and payment for services and acts as a passthrough for making API calls to the service.The daemon is the endpoint a client will submit requests to, and they are then passed to the service after validation by the daemon.
+
+
+## Treasure Server
```

Comment:
I think it is misleading to talk about "treasure server" when there is no such server yet. What we have is a command line tool to get a payment, update the channel state and write transaction to the blockchain. I would say at the moment it is a "channel claim cli".

Reply:
I would still propose to call it at least treasurer server logic. Otherwise people will not understand.

Reply:
Agree with VItaly, lets call it "Channel Claim command" for now

Reply:
fixed